store/sqlstore/group_store.go

```diff
@@ -1019,6 +1058,11 @@ func (s *SqlGroupStore) GetGroups(page, perPage int, opts model.GroupSearchOpts)
 		`, opts.NotAssociatedToChannel)
 	}
 
+	if opts.FilterAllowReference {
+		// Todo uncomment after column has been added
+		// 	groupsQuery = groupsQuery.Where("g.AllowReference = 1")
```

Comment:
need to be "g.AllowReference = true"

Reply:
I made that change in the feature branch